WORKSPACE

```diff
@@ -48,9 +48,16 @@ load("@aspect_rules_js//js:npm_import.bzl", "translate_pnpm_lock")
 
 translate_pnpm_lock(
     name = "npm",
-    pnpm_lock = "//examples:pnpm-lock.yaml",
+    pnpm_lock = "//:pnpm-lock.yaml",
 )
 
 load("@npm//:repositories.bzl", "npm_repositories")
 
 npm_repositories()
+
+load("@aspect_rules_swc//swc:repositories.bzl", "LATEST_VERSION", "swc_register_toolchains")
```

Comment:
We don't want to separate repo vs example dependencies?

Reply:
We do have https://github.com/aspect-build/bazel-examples/tree/main/ts_project_transpiler already.

I think our goal in each ruleset is minimal, standalone examples, while the bazel-examples repo is for larger ones that combine things.

I do agree that this is somewhere in the middle. What would you suggest for a minimal example of the transpiler attribute?

Reply:
I was mainly thinking about mixing repo vs example dependencies. Seeing this in the WORKSPACE makes me assume it is needed by rules_ts.

Reply:
yeah we've decided to keep the examples simple (and easily testable) by not making them nested workspaces.

Reply:
I'd just worry if we keep adding examples the WORKSPACE will become a mess.

Would a transpiler implemented in the example be more or less complicated? It means the example is self contained and doesn't pollute anything outside it such as this WORKSPACE, but means the example needs to somehow have it's own transpiler? Can it have a local package.json with a transpiler instead? Or a bzl with a dumb transpiler? 🤷